management/db.py

```diff
@@ -324,15 +327,19 @@ def is_owner(user_id,channel_id):
     Returns False if the user is not the owner, if the user doesn't exist or if the channel isn't found.
 
     Keyword arguments:
-    id -> the id of the user
-    channel -> the id of the channel"""
+    user_id -> the id of the user
+    channel_id -> the id of the channel"""
 
     # Check if the user exists
     c.execute("SELECT * FROM channel_rows WHERE id =?",(user_id,))
-    if c.fetchone() == None or check_for_int(user_id) == False or check_for_int(channel_id):
+    if c.fetchone() == None or check_for_int(user_id) == False or check_for_int(channel_id) == False:
+        print('Column doesn\'t exist!')
```

Comment:
Should this log to gamelog?

Reply:
We could, though we should catch this error on a higher level, as doing it right here would need the function to return a mailbox, most likely causing problems on higher levels as the output is to be distinguished.